en/docs/quick-starts/android.md

```diff
@@ -248,31 +308,30 @@ private void getUserInfo(){
                        mEmail = userInfoResponse.getUserInfoProperty("email");
                        JSONObject userInfoProperties = userInfoResponse.getUserInfoProperties();
                    }
-   
-                   if (mAuthenticationContext.getOAuth2TokenResponse() != null) {
-                       mIdToken = mAuthenticationContext.getOAuth2TokenResponse().getIdToken();
-                       mAccessToken = mAuthenticationContext.getOAuth2TokenResponse().getAccessToken();
-                   }
     }
 ```
 
 ### Logout
 
+- Have a button with id logout.
 - Call the logout method when logout button is clicked.
 
 ```java
-findViewById(R.id.logout).setOnClickListener(v -> Logout());
+findViewById(R.id.logout).setOnClickListener(v -> logout());
 
 ```
+- Call the logout method of LoginService instance.
 
 ```java
-private void Logout() {
+private void logout() {
 
         mLoginService.logout(this, mAuthenticationContext);
         finish();
     }
-```  
-  
+```
+
+## Sample app
+A sample is contained in this [repository](https://github.com/wso2-extensions/identity-samples-android.git)
```

Comment:
Refer this [repository](https://github.com/wso2-extensions/identity-samples-android.git) for a sample.